File: Backend/TrackrAPI/TrackrAPI/Dtos/GestionExpediente/ExpedienteWrapper.cs
using TrackrAPI.Dtos.Seguridad;
using TrackrAPI.Models;

namespace TrackrAPI.Dtos.GestionExpediente
{
    public class ExpedienteWrapper
    {
        public UsuarioDto paciente { get; set; }
        public ExpedienteTrackr expediente { get; set; }
        public Domicilio domicilio { get; set; }

        public IEnumerable<ExpedientePadecimientoDTO> padecimientos { get; set; }
    }
}

File: Frontend/trackr-paciente/src/app/shared/Dtos/seguridad/expediente-wrapper.ts
import { Usuario } from "@models/usuario";
import { ExpedientePadecimientoDTO } from '../../../../../../trackr-administrador/src/app/shared/dtos/seguridad/expediente-padecimiento-dto';
import { Domicilio } from '../../../../../../trackr-administrador/src/app/shared/models/seguridad/domicilio';
import { ExpedienteTrackR } from '../../../../../../trackr-administrador/src/app/shared/models/seguridad/expediente-trackr';

export class ExpedienteWrapper{
    expediente: ExpedienteTrackR;
    paciente: Usuario;
    domicilio: Domicilio;
    padecimientos: ExpedientePadecimientoDTO[];

    constructor(){
        this.expediente = new ExpedienteTrackR();
        this.paciente = new Usuario();
        this.domicilio = new Domicilio();
        this.padecimientos = [];
    }
}